src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -167,7 +170,9 @@
                                         :disabled?           (not valid-ens-or-address?)
                                         :on-press            #(rf/dispatch
                                                                [:wallet/select-send-address
-                                                                {:address @input-value
+                                                                {:address (if local-suggestion-address
```

Comment:
You can `or` the conditions instead of using `if`. A bit shorter.

Reply:
DOne